dash/App/Tokens/index.js

```diff
@@ -9,11 +9,12 @@ import CustomTokens from './CustomTokens'
 class Tokens extends React.Component {
   render () {
     const { notify } = this.props.data
-    console.log('this.props', this.props)
+    const activeChains = Object.values(this.store('main.networks.ethereum')).filter((chain) => chain.on)
```

Comment:
do we want to use "connected" chains here? also just had a thought: what if users want to add tokens for chains theyre not currently using?

Reply:
Asked Jordan about this one, a bit of an edge case and we can't autofill token details for chains which aren't active...